components/services/bat_ledger/bat_ledger_impl.cc

```diff
@@ -369,4 +369,26 @@ void BatLedgerImpl::AdSustained(const std::string& info) {
   ledger_->AdSustained(info);
 }
 
+void BatLedgerImpl::GetAdsNotificationsHistory(
+    const uint64_t from_timestamp_seconds,
+    const uint64_t to_timestamp_seconds,
+    GetAdsNotificationsHistoryCallback callback) {
+  auto* holder = new CallbackHolder<GetAdsNotificationsHistoryCallback>(
+      AsWeakPtr(), std::move(callback));
+
+  ledger_->GetAdsNotificationsHistory(from_timestamp_seconds,
+      to_timestamp_seconds, std::bind(
+      BatLedgerImpl::OnGetAdsNotificationsHistory, holder, _1));
+}
+
+// static
+void BatLedgerImpl::OnGetAdsNotificationsHistory(
```

Comment:
this one should be above `GetAdsNotificationsHistory`

Reply:
👍 done